src/main/webapp/download/index.jsp

```diff
@@ -313,6 +313,7 @@
     <li><a href="https://github.com/admc/wd">Javascript</a> by Adam Christian</li>
     <li><a href="https://github.com/jlipps/yiewd">Javascript</a> by Jonathan Lipps</li>
     <li><a href="http://webdriver.io">Javascript</a> by Camilo Tapia, Vincent Voyer and Christian Bromann</li>
+    <li><a href="https://theintern.github.io/leadfoot/">Leadfoot</a> by SitePen</li>
```

Comment:
"Leadfoot" isn't the language would rather it say Javascript in the link, you could put "Leadfoot by SitePen" after the link?